lib/utils.go

```diff
@@ -91,9 +97,23 @@ func GetRole(roleList saml.AssumableRoles, profileARN string) (saml.AssumableRol
 		return roleList[0], nil
 	}
 
+	// Sort the roles and group by account
+	sort.Slice(roleList, func(i, j int) bool {
+		return roleList[i].Role < roleList[j].Role
+	})
+
+	var previousAccountID, currentAccountID string
+
 	for i, arole := range roleList {
-		fmt.Printf("%d - %s\n", i, arole.Role)
+		currentAccountID = accountIDFromRoleARN(arole.Role)
+		if currentAccountID != previousAccountID {
+			fmt.Printf("\nAccount: %s\n", currentAccountID)
```

Comment:
This account header feels a little redundant given you print the whole role ARN each time anyway?

Reply:
You are right, it is redundant. I'll update it.